Mathlib/SetTheory/Ordinal/FixedPoint.lean

```diff
@@ -171,12 +171,15 @@ theorem derivFamily_strictMono [Small.{u} ι] (f : ι → Ordinal.{u} → Ordina
 
 theorem derivFamily_fp [Small.{u} ι] {i} (H : IsNormal (f i)) (o : Ordinal) :
     f i (derivFamily f o) = derivFamily f o := by
-  induction' o using limitRecOn with o _ o l IH
-  · rw [derivFamily_zero]
+  induction o using limitRecOn with
+  | H₁ =>
```

Comment:
This looks like an induction principle that needs its arguments renaming (in a pre-work PR)

Reply:
#23686